.github/workflows/build.yaml

```
@@ -0,0 +1,27 @@
+
+      - name: copy pdf files to build dir
+        run: |
+          mkdir -p build
+          cp *.pdf build/
+
+      - name: Push
+        uses: s0/git-publish-subdir-action@develop
+        env:
+          REPO: self
+          BRANCH: typst-build
+          FOLDER: build 
```

yamllint

[error] trailing spaces

(trailing-spaces)